package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -1121,9 +1128,11 @@ def _sameMolecTandDT(self, selection, t0d, tf):
 
     def _selection_serial(self, universe, selection_str):
         selection = []
-        for ts in ProgressBar(universe.trajectory, verbose=True,
-                              total=universe.trajectory.n_frames):
+        pm = ProgressMeter(universe.trajectory.n_frames,
+                           interval=10, verbose=True)
+        for ts in universe.trajectory:
             selection.append(universe.select_atoms(selection_str))
+            pm.echo(ts.frame)
```

Comment:
ProgressMeter should not be here. It's been deprecated and removed everywhere as part of PR #2617 .

You seem to be reverting PR #2617!